cogs/clash_river_war.py

```diff
@@ -61,6 +64,43 @@ def boat_attackers_embed(boat_attackers, clan_info):
 
     return embed
 
+def clan_river_race_history_embed(clan_players_war_history: Dict[str, PlayerActivity]):
+    embed = discord.Embed(
+        description=dedent('''
+            Clan River Race History
+        '''.format()),
+        colour=discord.Colour.green()
+    )
+    # columns = 'War Active | Elder Worthy | Name | Role | Level | Fame History | Boat Attack History | Average Fame'
+    # row_values = []
+    # for player_tag in clan_players_war_history:
+    #     player = clan_players_war_history[player_tag]
+    #     row_val = '` {} `|` {} `|` {} `|` {} `|` {} `|` {} `|` {} `|` {} '.format(
+    #         player.war_active, player.elder_worthy, player.name, player.role , player.exp_level , player.fame_hist, player.boat_attacks_hist, player.avg_fame
+    #     )
+    #     row_values.append(row_val)
+    columns = '**WarActive** | **ElderWorthy** | **FameHistory** | **Name** | **Role**'
+    row_promote = []
+    row_demote = []
+    row_final = []
+    for player_tag in clan_players_war_history:
+        player = clan_players_war_history[player_tag]
+        row_val = '` {:^1} ` | ` {:^1} ` | ` {} ` | ` {:>} ` | **{:>}**'.format(
+            'Y' if player.war_active else 'N',
+            'Y' if player.elder_worthy else 'N',
+            player.fame_hist,
+            player.role,
+            player.name
+        )
+        if player.war_active and player.elder_worthy and player.role == 'member':
+            row_promote.append(row_val)
+        elif not player.war_active:
+            row_demote.append(row_val)
+    
+    row_final = ['------------------------------- TO PROMOTE -------------------------------'] + row_promote + ['------------------------------- TO DEMOTE -------------------------------'] + row_demote

```

Comment:
ONLY for demotions section. Can you change it to `CONSIDER DEMOTE/KICK (Discuss in discord first)`